src/service/FileTransferService.ts

```diff
@@ -48,4 +52,33 @@ export class FileTransferService {
                 }
             });
     }
+
+    public async delete(fileId: string): Promise<void> {
+        if (fileId == null) {
+            throw new Error('File ID is missing');
+        }
+
+        const config: AxiosRequestConfig = {
+            headers: {
+                ...this.prepareHeaders()
+            }
+        };
+
+        return axios
+            .delete(`${this.url}/${fileId}`, config)
+            .then(() => {
+                return
+            }).catch((err) => {
+                if (err.response.status === NOT_FOUND) {
+                    throw new Error(`File ${fileId} cannot be deleted because it does not exist`)
+                }
+                throw new Error(`File ${fileId} cannot be deleted due to error: ${(err.message || 'unknown error').toLowerCase()}`)
+            });
+    }
+
+    private prepareHeaders(): Record<string, string> {
+        return {
+            'x-api-key': this.key
+        }
+    }
```

Comment:
Considering that the request only takes one header and it is unlikely that the API will be changed, why do we need this extra method?

Reply:
To cut on repetition.